bin/common.ml

```diff
@@ -515,7 +515,7 @@ module Builder = struct
     ; no_print_directory : bool
     ; store_orig_src_dir : bool
     ; default_target : Arg.Dep.t (* For build & runtest only *)
-    ; watch : Watch_mode_config.t
+    ; watch : Dune_rpc_impl.Watch_mode_config.t
```

Comment:
I looked around for a place to put `Watch_mode_config` that both `bin` executable and the `dune_rpc_impl` could share, but could not find any suitable. The best would be `dune_config_file` but `dune_rpc_impl` doesn't depend on it.

Reply:
That's fine. Moving stuff out of the binary is usually fine.